components/endorsal/actions/create-contact/create-contact.mjs

```diff
@@ -0,0 +1,121 @@
+import { ConfigurationError } from "@pipedream/platform";
+import { parseObj } from "../../common/utils.mjs";
+import endorsal from "../../endorsal.app.mjs";
+
+export default {
+  key: "endorsal-create-contact",
+  name: "Create Contact",
+  description: "Creates a new contact for requesting testimonials. [See the documentation](https://developers.endorsal.io/docs/endorsal/b3a6mtczmzu5na-create-a-new-contact)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    endorsal,
+    campaignID: {
+      propDefinition: [
+        endorsal,
+        "campaignID",
+      ],
+    },
+    name: {
+      propDefinition: [
+        endorsal,
+        "name",
+      ],
+      optional: true,
+    },
+    email: {
+      propDefinition: [
+        endorsal,
+        "email",
+      ],
+      optional: true,
+    },
+    phone: {
+      type: "string",
+      label: "Phone",
+      description: "The phone of the contact. `Email OR phone is required`.",
```

Comment:
If this code block is to be used, I think it'd be better to wrap only the props, such as:
```suggestion
      description: "The phone of the contact. `Email` OR `Phone` is required.",
```